client/src/pages/Items.js

```diff
@@ -1,102 +1,83 @@
+  },
+  {
+    name: 'Items (with only styled-components)',
+    toPathname: `${routes.ITEMS}/items-plain`,
+    pageComponent: ItemsPlain,
+  },
 ];
 
 class Items extends Component {
-    render() {
-
-        // TODO: would be better to dynamically create the routes based on page variations
-        const itemsPages = (
-            <Switch>
-                <Route exact path={routes.ITEMS} component={ItemsList} />
-                <Route exact path={`${routes.ITEMS}/react-table-v6`} component={ItemsTable} />
-                <Route exact path={`${routes.ITEMS}/items-plain`} component={ItemsPlain} />
-            </Switch>
-        );
-
-        return (
-            <Router>
-                <LinksGridContainer>
-                    {itemsPageVariants.map((itemsPageVariant, i) =>
-                        <LinkGridWrapper
-                            key={itemsPageVariant.name}
-                            style={{ gridColumn: `${((i + 2) * 2) - 1} / span 2` }}
-                        >
-                            <Button
-                                className="bg-dark"
-                                variant="contained"
-                            >
-                                <Link
-                                    style={{ color: linkTextColor(itemsPageVariant.toPathname) }}
-                                    to={itemsPageVariant.toPathname}
-                                >
-                                    {itemsPageVariant.name}
-                                </Link>
-                            </Button>
-                        </LinkGridWrapper>
-                    )}
-                </LinksGridContainer>
-                {itemsPages}
-            </Router>
-        );
-    };
-};
-
-const mapStateToProps = state => ({ ...state });
+  render() {
+    // TODO: would be better to dynamically create the routes based on page variations
+    const itemsPages = (
+      <Switch>
+        <Route exact path={routes.ITEMS} component={ItemsList} />
+        <Route exact path={`${routes.ITEMS}/react-table-v6`} component={ItemsTable} />
+        <Route exact path={`${routes.ITEMS}/items-plain`} component={ItemsPlain} />
+      </Switch>
+    );
 
-const mapDispatchToProps = dispatch => bindActionCreators(actions, dispatch);
+    return (
+      <>
+        <LinksGridContainer>
+          {itemsPageVariants.map((itemsPageVariant, i) => (
+            <LinkGridWrapper
+              key={itemsPageVariant.name}
+              style={{ gridColumn: `${(i + 2) * 2 - 1} / span 2` }}>
+              <Button className="bg-dark" variant="contained">
+                <Link
+                  style={{ color: linkTextColor(itemsPageVariant.toPathname) }}
+                  to={itemsPageVariant.toPathname}>
+                  {itemsPageVariant.name}
+                </Link>
+              </Button>
+            </LinkGridWrapper>
+          ))}
+        </LinksGridContainer>
+        {itemsPages}
+      </>
```

Comment:
Huh, maybe I just assumed I needed `Router` before and didn't even try using a fragment? Good catch! 🙂

Reply:
Nested Browser Routes will cause some route issues when navigating pages, i just remove it